CHANGELOG.md

```diff
@@ -4,7 +4,8 @@
 
 ### Fixes
 
-- Fix: Only call method channels on native platforms ([#1196](https://github.com/getsentry/sentry-dart/pull/1196))
```

Comment:
Why is this one deleted?

Reply:
Only removed the "fix" prefix.

Reply:
Its not deleted, just removed the `Fix:` cus its already under the `### Fixes`.